iiwa_cam/GeneralControl general_control
---
bool success
string error_msg
iiwa_cam/GeneralTraj general_traj